### PR TITLE
Add support to have custom events on a thumbs-view.

### PR DESCRIPTION
**Pull request by andymoon**
This allows you to use custom events on your views that are added as thumbs-view and still use thumbs-delegate to delegate the custom events.  They no long have to be dom events.

example.

<span data-thumbs-view="Button" data-thumbs-delegate="onClick:execute"></span>

### CODE REVIEW

File: dist/thumbs.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
```
Comment (nicknisi) [inline dist/thumbs.js]: I don't think this file should exist.

**Comment by nicknisi** [comment]
Where are the tests?

**Comment by andymoon** [comment]
Added some tests.

**Comment by nicknisi** [comment]
Looks good.

**Pull request merged**